docs/environments/third_party_environments.md

```diff
@@ -97,6 +97,13 @@ goal-RL ([Gymnasium-Robotics](https://robotics.farama.org/)).
 ### Game environments
 *Board Games, Video Games and all other interactive entrainment mediums.*
 
+- [Generals.io bots: Develop your agent for generals.io!](https://github.com/strakam/generals-bots)
```

Comment:
Could you add this in alphabetical order?

Reply:
Done